bentoctl/cli/operator_management.py

```diff
@@ -127,7 +126,7 @@ def remove(name, skip_confirm):  # pylint: disable=unused-variable
     @click.argument("name")
```

Comment:
Let's add an optional version argument. If not specified, the command will update to the latest.